dbms/src/Storages/S3/S3RandomAccessFile.cpp

```diff
@@ -192,7 +192,16 @@ bool S3RandomAccessFile::initialize()
         auto outcome = client_ptr->GetObject(req);
         if (!outcome.IsSuccess())
         {
-            LOG_ERROR(log, "S3 GetObject failed: {}, cur_retry={}", S3::S3ErrorMessage(outcome.GetError()), cur_retry);
+            auto el = sw.elapsedSeconds();
+            LOG_ERROR(
+                log,
+                "S3 GetObject failed: {}, cur_retry={}, key={}, remote_fname={}, elapsed{}={}",
+                S3::S3ErrorMessage(outcome.GetError()),
+                cur_retry,
+                req.GetKey(),
```

Comment:
Is there anywhere `key_root` is printed?

Reply:
Then I think we'd better also print it here, since start log could lose when a server is starting. Meanwhile, it could be hard to find if the server has been running for a long time.